models/topics.models.js

```diff
@@ -1,4 +1,12 @@
 const db = require("../db/connection");
+const { readFile } = require("fs/promises");
+const file = `${__dirname}/../endpoints.json`;
+
+exports.fetchApis = () => {
+  return readFile(file, "utf-8").then((apis) => {
+    return JSON.parse(apis);
```

Comment:
this is fine.  But you could just require the JSON endpoints file in your controller and sending it as your response, instead of reading a file in your model.

Reply:
Noted. Thanks!